pom.xml

```diff
@@ -562,6 +579,76 @@
                 <artifactId>commons-codec</artifactId>
                 <version>1.15</version>
             </dependency>
+
```

Comment:
Maybe you could add a comment above this dependencyManagement block to say what it is?

Reply:
I am honestly not sure what this is and how it got here because it's not an indirect dependency unless it was one of some package that we stopped using recently…

Reply:
Ah, turns out this is a dependency of RESTEasy and it looks like there's no need to pin the version. Not sure why it ended up in the `pom.xml` file.